lib/capistrano/datadog.rb

```diff
@@ -13,7 +13,9 @@ def self.reporter()
 
     def self.cap_version()
       if @cap_version.nil? then
-        if Configuration.respond_to? :instance then
+        if !defined? Configuration
```

Comment:
`Configuration` is not defined at test time, library would eventually fail with an empty version but for testing is ok.